docs/framework/20categories/40operation/ability-pause.md

```diff
@@ -1,4 +1,21 @@
 ### Ability to Pause
-If pausing is controlled by a multi-sig, then there is the risk 
-that not enough parties can be gathered quickly enough to pause 
-the project.
+The pausing capability described in the [Ability to Pause Implementation Risk section](../30implementation/pause.md)
+is only effective if the ```pause()``` function can be executed expeditiously. Typically, attacks on protocols are mounted in time periods ranging from minutes to hours. As such, pausing a project several hours after an attack has commenced 
+is unlikely to effective, as by this time, the project may have already been drained of funds.
+
+To prevent malicious parties unnecessarily halting the project by calling the ```pause()``` function, ```pause()``` functions need to have some form of [access control](../30implementation/access-control.md). 
+
+For projects operated by a single organization, a simple administration set-up could be used. However, as described below, this has serious issues. For example, a single administrator account (that is an *owner* account) could be the only party authorized to call the ```pause()``` function. However, to be responsive to the need to pause the project, the private key belonging to the administrator account would need to be shared with support staff who live in time zones around the world. The issue with sharing a single private key is that it would be impossible to determine which support staff member used the key to pause the project. Additionally, if one of the support staff left the company or were compromised, then the shared key would need to be changed.
```

Comment:
nit: consistency in using either "time zone", "timezone", or "time-zone"

Reply:
Going for one word.